salt/cert/init.sls

```diff
@@ -16,7 +16,12 @@ include:
   {% endfor %}
 
   # add some extra names the API server could have
-  {% set extra_names = extra_names + ["DNS: api",
+  {% set extra_names = extra_names + ["DNS: kubernetes",
+                                      "DNS: kubernetes.default",
+                                      "DNS: kubernetes.default.svc",
+                                      "DNS: kubernetes.default.svc.cluster",
+                                      "DNS: kubernetes.default.svc.cluster.local",
```

Comment:
I've just realized we are adding those names for `pillar["dns:domain"]` [here](https://github.com/kubic-project/salt/blob/b48feae530cbd51ef6ca4c00903893cada020eb7/salt/cert/init.sls#L38). I guess can remove these `cluster.local` entries...